### ISSUE TITLE
Predict synonimous mutations

### ISSUE THREAD

**Issue by dgruano** [pydna-group/pydna#267]
I have mainly used CRISPR/Cas9 to engineer point mutations in yeast. These are hard to follow after crossing, and we usually added synonimous point mutations that would add or remove a restriction site. This made genotyping of strains straightforward with PCR + digestion. We also used this to mutate the PAM sequence and avoid any further Cas9 cuts after the desired point mutation was integrated.

I wonder whether a function to find possible synonimous mutations could be a good addition to pydna. I already have a couple of scripts with basic functionality and would be happy to integrate them if you think it's useful.

What are your views on this? @BjornFJohansson @manulera @hiyama341

**Comment by BjornFJohansson** [comment]
I think this is a nice idea. Perhaps we could discuss how this should be presented to the user.
